core/shared/src/main/scala/zio/Chunk.scala

```diff
@@ -597,6 +597,11 @@ sealed trait Chunk[+A] { self =>
 
 object Chunk {
 
+  implicit class ChunkOps[A](private val self: Chunk[A]) extends AnyVal {
```

Comment:
Don't need the `A1` type parameter here since `A` in `ChunkOps` is invariant.

Reply:
Not sure about that, it doesn't create compilation error in ZIO, but when with use subtyping, doesn't work.

```scala
trait A
trait B extends A
val x: Chunk[B] = Chunk.empty
//x: zio.Chunk[B] = Empty$()
x ++ (new A {})
<console>:28: error: overloaded method value ++ with alternatives:
  (nonEmptyChunk: zio.NonEmptyChunk[B])zio.NonEmptyChunk[B] <and>
  (chunk: zio.Chunk[B])zio.Chunk[B]
 cannot be applied to (A)
       x ++ (new A {})
```